homeassistant/components/todoist/strings.json

```
@@ -28,6 +28,14 @@
       "message": "Project \"{project}\" has no section \"{section}\""
     }
   },
+  "selector": {
+    "reminder_location_direction": {
+      "options": {
+        "on_enter": "Entering",
+        "on_leave": "Leaving"
+      }
+    }
+  },
   "services": {
     "new_task": {
       "name": "New task",
```

Action names should use a verb:

```suggestion
      "name": "Create task",
```
Also it seems that the "description" string is missing, something like "Adds a new task plus optional reminder to …" – you'll know best how to describe it.